### PR TITLE
add 3D GameStudio

### LINKED ISSUE #70: Engine: 3D Gamestudio aka Gamestudio A8
### Some reference to the engine, such as the developer site or a wikipedia article.

3D Gamestudio, oftentimes only referred to as Gamestudio, is a very old school toolset (engine + editors) which some people used before Unity was a thing. It might be referred to as A8 as well, short for Acknex 8 (rolls right of the tongue, right?).

Super old school homepage: http://www.conitec.net/english/gstudio/index.PHP
Manual and API reference: http://manual.3dgamestudio.net/

It's a Windows only 32bit DirectX 9.0c engine. It's hell to work with.

### SteamDB links

(My game: ) https://steamdb.info/app/688130/

https://steamdb.info/app/724390/

https://steamdb.info/app/34900/

https://steamdb.info/app/259640/

https://steamdb.info/app/280640/

Some of those are made with previous iterations of Gamestudio (A4 through A8) but they all contain the same base data.

### Possible way to detect it

The engine DLL **acknex.dll** should be included in pretty much every game made with this toolset.
*.wrs container files are present if the game creator owns the Pro version of the engine.
A file called acknex.wdf, a starter window definition, might be present, as well as d3dx9_42.dll and/ or d3dx9_32.dll and d3dx9_30.dll files. Furthermore, d3dcompiler_42.dll might be included in most games.

### PR DESCRIPTION
**Pull request by diedyesterdaywastaken**
### SteamDB app page links to a few games using this

- [AaaaaAAaaaAAAaaAAAAaAAAAA!!! - A Reckless Disregard for Gravity](https://steamdb.info/app/15520/)
- [Bad Rats: The Rats' Revenge](https://steamdb.info/app/34900/)
- [Dark Shadows: Army of Evil](https://steamdb.info/app/280640/)
- [Guise of the Wolf](https://steamdb.info/app/259640/)
- [The Wonderful End of the World](https://steamdb.info/app/15500/)

### Brief explanation of the change

closes #70

### CODE REVIEW

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.3D\_GameStudio**

These apps now match: **(54)** [15500](https://steamdb.info/app/15500/depots/) [15510](https://steamdb.info/app/15510/depots/) [15520](https://steamdb.info/app/15520/depots/) [15530](https://steamdb.info/app/15530/depots/) [34900](https://steamdb.info/app/34900/depots/) [34910](https://steamdb.info/app/34910/depots/) [34920](https://steamdb.info/app/34920/depots/) [34930](https://steamdb.info/app/34930/depots/) [209060](https://steamdb.info/app/209060/depots/) [259640](https://steamdb.info/app/259640/depots/) [273770](https://steamdb.info/app/273770/depots/) [280640](https://steamdb.info/app/280640/depots/) [336200](https://steamdb.info/app/336200/depots/) [339720](https://steamdb.info/app/339720/depots/) [353020](https://steamdb.info/app/353020/depots/) [380950](https://steamdb.info/app/380950/depots/) [412290](https://steamdb.info/app/412290/depots/) [450030](https://steamdb.info/app/450030/depots/) [459510](https://steamdb.info/app/459510/depots/) [502720](https://steamdb.info/app/502720/depots/) [507390](https://steamdb.info/app/507390/depots/) [512150](https://steamdb.info/app/512150/depots/) [513650](https://steamdb.info/app/513650/depots/) [546780](https://steamdb.info/app/546780/depots/) [553100](https://steamdb.info/app/553100/depots/) [587330](https://steamdb.info/app/587330/depots/) [670760](https://steamdb.info/app/670760/depots/) [688130](https://steamdb.info/app/688130/depots/) [691700](https://steamdb.info/app/691700/depots/) [714210](https://steamdb.info/app/714210/depots/) [724390](https://steamdb.info/app/724390/depots/) [747610](https://steamdb.info/app/747610/depots/) [761510](https://steamdb.info/app/761510/depots/) [782160](https://steamdb.info/app/782160/depots/) [793580](https://steamdb.info/app/793580/depots/) [807000](https://steamdb.info/app/807000/depots/) [844710](https://steamdb.info/app/844710/depots/) [846140](https://steamdb.info/app/846140/depots/) [846850](https://steamdb.info/app/846850/depots/) [977090](https://steamdb.info/app/977090/depots/) [1040450](https://steamdb.info/app/1040450/depots/) [1051840](https://steamdb.info/app/1051840/depots/) [1104590](https://steamdb.info/app/1104590/depots/) [1189240](https://steamdb.info/app/1189240/depots/) [1315890](https://steamdb.info/app/1315890/depots/) [1317800](https://steamdb.info/app/1317800/depots/) [1423980](https://steamdb.info/app/1423980/depots/) [1585250](https://steamdb.info/app/1585250/depots/) [1768900](https://steamdb.info/app/1768900/depots/) [1845670](https://steamdb.info/app/1845670/depots/) [2100790](https://steamdb.info/app/2100790/depots/) [2155700](https://steamdb.info/app/2155700/depots/) [2942450](https://steamdb.info/app/2942450/depots/) [2942480](https://steamdb.info/app/2942480/depots/)

Please confirm that all these changed apps are correct.

**Comment by Lifeismana** [comment]
why aren't you linking to the official website ? <http://www.3dgamestudio.com/>
Non matching tests missing

**Comment by diedyesterdaywastaken** [comment]
okay i think i fixed everything

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.3D\_GameStudio**

These apps now match: **(54)** [15500](https://steamdb.info/app/15500/depots/) [15510](https://steamdb.info/app/15510/depots/) [15520](https://steamdb.info/app/15520/depots/) [15530](https://steamdb.info/app/15530/depots/) [34900](https://steamdb.info/app/34900/depots/) [34910](https://steamdb.info/app/34910/depots/) [34920](https://steamdb.info/app/34920/depots/) [34930](https://steamdb.info/app/34930/depots/) [209060](https://steamdb.info/app/209060/depots/) [259640](https://steamdb.info/app/259640/depots/) [273770](https://steamdb.info/app/273770/depots/) [280640](https://steamdb.info/app/280640/depots/) [336200](https://steamdb.info/app/336200/depots/) [339720](https://steamdb.info/app/339720/depots/) [353020](https://steamdb.info/app/353020/depots/) [380950](https://steamdb.info/app/380950/depots/) [412290](https://steamdb.info/app/412290/depots/) [450030](https://steamdb.info/app/450030/depots/) [459510](https://steamdb.info/app/459510/depots/) [502720](https://steamdb.info/app/502720/depots/) [507390](https://steamdb.info/app/507390/depots/) [512150](https://steamdb.info/app/512150/depots/) [513650](https://steamdb.info/app/513650/depots/) [546780](https://steamdb.info/app/546780/depots/) [553100](https://steamdb.info/app/553100/depots/) [587330](https://steamdb.info/app/587330/depots/) [670760](https://steamdb.info/app/670760/depots/) [688130](https://steamdb.info/app/688130/depots/) [691700](https://steamdb.info/app/691700/depots/) [714210](https://steamdb.info/app/714210/depots/) [724390](https://steamdb.info/app/724390/depots/) [747610](https://steamdb.info/app/747610/depots/) [761510](https://steamdb.info/app/761510/depots/) [782160](https://steamdb.info/app/782160/depots/) [793580](https://steamdb.info/app/793580/depots/) [807000](https://steamdb.info/app/807000/depots/) [844710](https://steamdb.info/app/844710/depots/) [846140](https://steamdb.info/app/846140/depots/) [846850](https://steamdb.info/app/846850/depots/) [977090](https://steamdb.info/app/977090/depots/) [1040450](https://steamdb.info/app/1040450/depots/) [1051840](https://steamdb.info/app/1051840/depots/) [1104590](https://steamdb.info/app/1104590/depots/) [1189240](https://steamdb.info/app/1189240/depots/) [1315890](https://steamdb.info/app/1315890/depots/) [1317800](https://steamdb.info/app/1317800/depots/) [1423980](https://steamdb.info/app/1423980/depots/) [1585250](https://steamdb.info/app/1585250/depots/) [1768900](https://steamdb.info/app/1768900/depots/) [1845670](https://steamdb.info/app/1845670/depots/) [2100790](https://steamdb.info/app/2100790/depots/) [2155700](https://steamdb.info/app/2155700/depots/) [2942450](https://steamdb.info/app/2942450/depots/) [2942480](https://steamdb.info/app/2942480/depots/)

Please confirm that all these changed apps are correct.

File: descriptions/Engine.3D_GameStudio.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[**3D GameStudio**](http://www.3dgamestudio.com/) or **3DGS** is a pan 3D computer game development system which allows the users to create 3D games and other virtual reality applications, and publish them royalty-free.
```
Comment (Lifeismana) [inline descriptions/Engine.3D_GameStudio.md:1]:
```suggestion
[**3D GameStudio**](http://www.3dgamestudio.com/) is a set of tools to create 3D games
```
no need to copy paste wikipedia's short description

**Pull request merged**